parlai/tasks/wizard_of_wikipedia/agents.py

```diff
@@ -907,6 +918,14 @@ class DefaultTeacher(WizardDialogKnowledgeTeacher):
     pass
 
 
+class SelfchatTeacher(WizardDialogKnowledgeTeacher):
```

Comment:
SelfChatTeacher?

Reply:
i think the intention originally of alex and i in parlai was that you could enter lowercase in commandline and it would work because it would uppercase the first letter:
https://github.com/facebookresearch/ParlAI/blob/master/parlai/core/worlds.py#L1147
but now we seem to have a smattering of both types (only first letter uppercase,  every word uppercase) because people didnt follow this..    note all the tasks are in lowercase as well

Reply:
so should we abandon this thing and try and make everything cased in future?  it's half and half right now..  or go back to the old way

Reply:
I think if it has an underscore the next letter becomes caps. I agree we should standardize

Reply:
You should do what you like here